examples/js/loaders/SVGLoader.js

```diff
@@ -221,14 +221,13 @@ THREE.SVGLoader.prototype = {
 						break;
 
 					case 'A':
-						console.warn( command );
 						var numbers = parseFloats( data );
 						for ( var j = 0, jl = numbers.length; j < jl; j += 7 ) {
-							// TODO
-							point.x = numbers[ j + 5 ];
```

Comment:
You shouldn't remove this. You'll break further relative commands otherwise.

Reply:
right, I corrected it in next commit

Reply:
Thanks! I'll merge and clean up.